index.html

```diff
@@ -37,13 +37,13 @@
 
       <form id="form" action="https://www.freecodecamp.com/email-submit">
         <input type="email" id="email" name="email" placeholder="enter email"/><br/>
-        <input type="submit" id="submit" formaction="https://www.freecodecamp.com/email-submit"/>
+        <input type="submit" id="submit"/>
       </form>
       <div class="Prices" id="Pricing">
           <div class="price_style">
-            <h3>
+            <div>
               TENOR TROMBONE
-             </h3>

```

Comment:
- [ ] You should use a `h1` tag for your titles, using a '<div>' for titles not semantic HTML

Reply:
A better way to show your opinion about an issue is to make suggestions 😊, instead of giving orders.😠👉
Also including an explanation, a "why", would benefits your review and adds more value to it.
 
_Example_

- This is a polite suggestion: **"I think it would be better to use semantic tags"**
- This reads like an order: **"Use semantic tags"**

_Example_
- This is a well explained request: **Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible**
- This request lacks a "why": **Why don't you use an article tag here instead of a div?**